nilearn/image/image.py

```diff
@@ -835,6 +836,10 @@ def clean_img(imgs, sessions=None, detrend=True, standardize=True,
     t_r: float, optional
         Repetition time, in second (sampling period).
 
+    ensure_finite: bool, optional
+        If True, the infinite values (NANs and infs) found in the images
+        will be replaced by zeros.
```

Comment:
First, it is not about "infinite" values but "non-finite" values (because NaN is not an infinity). Second, you rely on `np.nan_to_num` that set NaNs to 0, Inf to a very big value and NegInf to a very small value. So your doc is not correct.

Although we agree that NaN = 0 in this case, I don't know what we should do with infinities... Maybe raise an error?